lib/phlex/component.rb

```diff
@@ -50,12 +50,12 @@ def content(&block)
       output = yield(self) if block_given?
```

Comment:
I know this is unrelated, but this conditional isn't necessary. Could we remove it in this PR?

```suggestion
      output = yield(self)
```